config/env/development.js

```diff
@@ -18,4 +18,7 @@ module.exports = {
   // models: {
   //   connection: 'someMongodbServer'
   // }
+
+  // config/sockets.js
+  onlyAllowOrigins: undefined,
```

Comment:
We may not need this since it's explicitly everything when not set, but I wanted to be specific. Don't care if we drop it though.